articles/github-copilot-azure/TOC.yml

```
@@ -0,0 +1,26 @@
+    - name: Design and develop your application
+      href: design-develop-examples.md
+    - name: Deploy your application
+      href: deploy-examples.md
+    - name: Troubleshooting your application
+      href: troubleshoot-examples.md
+    - name: Optimize your application
+      href: optimize-examples.md
+    - name: Advanced scenarios
+      items: 
+        - name: Create Bicep files
+          href: bicep-generate-edit.md
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)